.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.vscode
```

Comment:
I added this as a first timer issue. Thought someone else who is a first timer could pick it up.

Reply:
got it, accidentally put it there. forgot about the first timer issue 😁